presto-main/src/main/java/com/facebook/presto/util/FastutilSetHelper.java

```diff
@@ -65,43 +66,140 @@ public static boolean in(boolean booleanValue, BooleanOpenHashSet set)
         return set.contains(booleanValue);
     }
 
-    public static boolean in(byte byteValue, ByteOpenHashSet set)
+    public static boolean in(double doubleValue, DoubleOpenCustomHashSet set)
     {
-        return set.contains(byteValue);
+        return set.contains(doubleValue);
     }
 
-    public static boolean in(char charValue, CharOpenHashSet set)
+    public static boolean in(long longValue, LongOpenCustomHashSet set)
     {
-        return set.contains(charValue);
+        return set.contains(longValue);
     }
 
-    public static boolean in(double doubleValue, DoubleOpenHashSet set)
+    public static boolean in(Object sliceValue, ObjectOpenHashSet<Slice> set)
```

Comment:
these helpers with object are not really needed in that that whole purpose is to avoid boxing primitives. In this case, the y are no longer primitives.